paasta_tools/cli/cmds/sysdig.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import shlex
+import subprocess
+import sys
+from urlparse import urlparse
+
+from paasta_tools.cli.utils import calculate_remote_masters
+from paasta_tools.cli.utils import find_connectable_master
+from paasta_tools.cli.utils import get_status_for_instance
+from paasta_tools.cli.utils import get_subparser
+from paasta_tools.cli.utils import pick_slave_from_status
+from paasta_tools.marathon_tools import load_marathon_config
+from paasta_tools.mesos_tools import get_mesos_master
+from paasta_tools.utils import _run
+from paasta_tools.utils import load_system_paasta_config
+
+
+def add_subparser(subparsers):
+    new_parser = get_subparser(description="'paasta sysdig' works by SSH'ing to remote PaaSTA masters and "
+                                           "running sysdig with the neccessary filters",
+                               help_text="Run sysdig on a remote host and filter to a service and instance",
+                               command='sysdig',
+                               function=paasta_sysdig,
+                               subparsers=subparsers)
+    new_parser.add_argument(
+        '-l', '--local',
+        help="Run the script here rather than SSHing to a PaaSTA master",
+        default=False,
+        action='store_true'
+    )
+
+
+def get_any_mesos_master(cluster):
+    system_paasta_config = load_system_paasta_config()
+    masters, output = calculate_remote_masters(cluster, system_paasta_config)
+    if masters == []:
```

Comment:
I'd use `if not masters:` here, which won't break if `calculate_remote_masters ` ever changes to return a set, tuple, generator, etc.

(Also it's the PEP8 approved way of checking if sequences are empty).